src/core/config/type-cast/type-casters/index.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * @prettier
```

Comment:
Could we move this to `config/type-cast/mappings.js` ? 

Reasoning - let's name the module of what it actually represents - mappings between paths and type casters